myapp/app/models/task.rb

```diff
@@ -0,0 +1,13 @@
+class Task < ApplicationRecord
+
+  STATUSES = ["Not Started", "In Progress", "Completed"].freeze
```

Comment:
You can use `enum`
ex
```
enum status: { not_started: 0, in_progress: 1, completed: 2 }
```
https://blog.saeloun.com/2022/01/05/how-to-use-enums-in-rails/
https://edgeapi.rubyonrails.org/classes/ActiveRecord/Enum.html

Reply:
modified